Mathlib/Probability/Kernel/Defs.lean

```diff
@@ -224,6 +225,7 @@ theorem ext_fun (h : ∀ a f, Measurable f → ∫⁻ b, f b ∂κ a = ∫⁻ b,
 theorem ext_fun_iff : κ = η ↔ ∀ a f, Measurable f → ∫⁻ b, f b ∂κ a = ∫⁻ b, f b ∂η a :=
   ⟨fun h a f _ => by rw [h], ext_fun⟩
 
+-- could be @[fun_prop] if fun_prop could synthesize the `MeasurableSet s` argument
```

Comment:
I'm not sure about these comments: fun_prop doesn't know about measurable sets. Should this rather be a library_note than a comment?